webapp/platform/client/src/client4.ts

```diff
@@ -1747,7 +1747,7 @@ export default class Client4 {
         );
     };
 
-    getChannelStats = (channelId: string, excludeFilesCount = false) => {
+    getChannelStats = (channelId: string, excludeFilesCount = true) => {
```

Comment:
This change prevents that a stats request from any other place different from the Channel Info RHS panel would accidentaly include the file count request. This is the right way since the file count is only used there and now will be conditioned by the config value.

Reply:
:+1: for this change. The query is expensive and hence should be opt-in.